src/java.desktop/share/classes/javax/imageio/plugins/tiff/BaselineTIFFTagSet.java

```diff
@@ -222,7 +222,8 @@ public final class BaselineTIFFTagSet extends TIFFTagSet {
      * A value to be used with the "Compression" tag.
      *
      * @see #TAG_COMPRESSION
-     * @see <a href="https://tools.ietf.org/html/rfc1951">DEFLATE specification</a>
+     * @spec https://www.rfc-editor.org/info/rfc1951
+     *      RFC 1951: DEFLATE Compressed Data Format Specification version 1.3
      */
```

Comment:
OK. So why are you preferring this rfc-editor.org site ?
Seems we should get that cleared up before I look further

Reply:
Because we were strongly recommended to do so, in an earlier round of review.

Reply:
I have no idea who that person is with the strong opinions, and hence no idea of his standing to assert the opinion.
So I'd be reluctant to follow his suggestion without more info.
But they still should be the same.